File: topics/03-http-server-router/java/src/test/java/dev/reinvent/wheel/t03/HttpServerRouterAppTest.java
package dev.reinvent.wheel.t03;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

final class HttpServerRouterAppTest {
    @Test
    void minimalTemplateCompilesAndRuns() {
        HttpServerRouterApp.CliArgs args = new HttpServerRouterApp.CliArgs(18081);
assertDoesNotThrow(() -> HttpServerRouterApp.start(args.port()).stop(0));
    }
}

File: topics/03-http-server-router/kotlin/src/test/kotlin/dev/reinvent/wheel/t03/HttpServerRouterAppTest.kt
package dev.reinvent.wheel.t03

import kotlin.test.Test

class HttpServerRouterAppTest {
    @Test
    fun minimalTemplateCompilesAndRuns() {
        val args = HttpServerRouterApp.CliArgs(18082)
HttpServerRouterApp.start(args.port).stop(0)
kotlin.test.assertTrue(true)
    }
}
